src/Libraries/Microsoft.Extensions.Http.Diagnostics/README.md

```diff
@@ -55,6 +55,47 @@ builder.Services.AddHttpClientLogEnricher<MyHttpClientLogEnricher>();
 var host = builder.Build();
 ```
 
+It is important to note that the `AddExtendedHttpClientLogging` method will add information to the logs using *enrichment*. This means that the information will be added as tags to the structured logs, but will not be visible in the log message that is printed by default in the console. To view the information, you will need to use a logging provider that supports structured logs. One quick and built-in way to do this, is to call `AddJsonConsole()` to your logging builder, which will print out the full structured logs to the console. Here is a quick sample that uses the `ExtendedHttpClientLogging()` method to automatically log all `HttpClient` request and response bodies, and then prints the full structured logs to the console:
```

Comment:
Small nit: the example below would only print the response body, because only `ResponseBodyContentTypes` is set up, right? And the description says "request and response bodies"

Reply:
Good catch! I'll fix that in a follow-up PR.